src/Footer.scss

```
@@ -0,0 +1,14 @@
+.footer-main {
+    display: flex;
+    flex-direction: row; // make the child elements display horizontally
+    justify-content: space-between; // distribute the child elements to the left and right edges
```

You don't need to comment these!
```suggestion
    flex-direction: row;
    justify-content: space-between;
```